### PR TITLE
Add 2-lines support for header

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] https://github.com/Samsung/TAU/issues/638
[Problem] No 2lines support in header
[Solution] Created css class with appropriate height in order to support 2 lines header

Signed-off-by: Lukasz Slachciak <l.slachciak@samsung.com>

### CODE REVIEW

File: src/css/profile/wearable/changeable/theme-circle/layout.less [hunk 1]
```diff
@@ -36,6 +36,9 @@ body {
 				margin-top: -7 * @unit_base;
 			}
 		}
+		&.ui-header-2-lines {
+			height: 124 * @unit_base;
```
Comment (TomaszLukawskiSam) [inline src/css/profile/wearable/changeable/theme-circle/layout.less:40]: add: margin-top: 25 * @unit_base

**Review by TomaszLukawskiSam — CHANGES_REQUESTED** [review]

File: src/css/profile/wearable/changeable/theme-circle/layout.less (hunk 1 above)
Comment (lslachciak) [reply src/css/profile/wearable/changeable/theme-circle/layout.less:40]: done

**Comment by lslachciak** [comment]
Fixed according to the @TomaszLukawskiSam comment
Also removed media queries

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

**Pull request merged**